samza-core/src/test/java/org/apache/samza/task/TestAsyncRunLoop.java

```diff
@@ -368,8 +367,10 @@ public void testCommitSingleTask() throws Exception {
     task0ProcessedMessagesLatch.await();
     task1ProcessedMessagesLatch.await();
 
-    verify(offsetManager).checkpoint(taskName0);
-    verify(offsetManager, never()).checkpoint(taskName1);
+    verify(offsetManager).createCheckpoint(taskName0);
```

Comment:
Agree, but to fix this we'd have to convert all TaskInstances in this test suite to be mocks